frontend/views/Login.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from "react";
+import { LinearGradient } from "expo-linear-gradient";
+import { Alert, View, StyleSheet } from "react-native";
+import { CommonActions } from "@react-navigation/native";
+import { Button, Card, TextInput } from "react-native-paper";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+
+import { Admin } from "../api/admin";
+import { updateUser } from "../redux/UserAction";
+
+const styles = StyleSheet.create({
+  card: {
+    marginTop: 100,
+    marginLeft: 20,
+    marginRight: 20,
+  },
+  container: {
+    padding: 10,
+    fontSize: 18,
+  },
+});
+
+function Login({ route, navigation, updateUser, user }) {
```

Comment:
I think it's a good idea to follow a similar format to how Culture implements Props so that way we have type inference.

Specify props that are passed by a previous view in `frontend/routes.ts`.

Specify props like the navigator and others in [old diff](https://github.com/freeman91/OSUMC-Cultural-Awareness-App/pull/31/files#diff-537f24867236509ae2d1933ed9da20abed01c85db30ae136d9754f35f591e35dR34-R37)